modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/RubyClientCodegen.java

```diff
@@ -605,6 +612,13 @@ public String toEnumName(CodegenProperty property) {
         }
     }
 
+    @Override
+    protected void addImport(CodegenModel m, String type) {
+        if (m.parent != null && m.parent.equals(type)) {
+            super.addImport(m, type);
+        }
+    }
+
```

Comment:
Only adds the parent class for importing as the ruby client does not need to know the type of the values it works with, but does need to import the parent class